app/src/main/res/layout/header_item.xml

```diff
@@ -32,4 +32,16 @@
         android:gravity="center_vertical"
         tools:text="Game of Thrones" />
 
+    <TextView
+        android:id="@+id/header_more"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_marginEnd="14dp"
```

Comment:
There aren't any `14dp` or `16dp` dimens used yet (except in this file), but I'll put put them into a `dimens.xml` for future uses